### DISCUSSION TITLE
stop asking after rtsp:// feed ?

### DISCUSSION THREAD

**Discussion by JensJackowski** [frenck/spook#782] [Ideas]
I use many cameras in many integrations in HomeAssistent (camera integration) and in Frigate.

It is no problem to configure them on the fly, turn motion dectection / recording and other services "on and off" by simple switches usually provided by the integrations.

But i didn´t find a solution that stopp the HomeAssistant / Frigate / Camera integration to constantly try to receveice a provided rtsp:// feed.

I have a bunch of cheap cameras that does not more than just provide an rtsp:// feed - and if "noone" asking after this they will enter a energy saving "sleep mode".

Usecase are cameras that only have a meaning under special conditions. It is for example verry boring to look att a 3D-printer building plate when the printer is not working.

When i discuss this topic in the community i got the impression that many people have such an usecase and that their is no easy solution for this. Same are using Spook for turning the whole integration on- and off or has found other "backdoors". And most likely i will find a way to do this when i put much effort in this question.

But anoying many people simple do not care and just accept that waste of ressources that the unnesserary network-trafic and a camera that is awake the whole time generate.

So i tought it might be an interessting feature for samething like spook. 

And i will be glad if anyone can give me a hint how i can reach my goal easyly.

And yes: It can look a bit odd to care about ressources that are used for no real reasons but do not disturb. But att least it has an effect on energy consumtion and the enviromental impact. Many underestimate the enviromental impact because each waterdropp doesn´t seem important....

So i think it is worth to consider if their is a way to make it easyier to "do right".

And i think that the model "camera is awake if anyone asking for a stream" is a verry commen design. It works good for cameras that are deisgned to use with an app that only connects to the camera if anyone opens the app and ask for a stream.

**Comment by frenck** [comment]
Disable the device or entity when you don't need it?

spook provide services for those

**Reply by JensJackowski** [reply]
yes i know, but i end up in a loop.

For example: If i manualy disable devices that are speciefied in frigate.yalm it will generate faults that i can stop with spook. On the next restart of HomeAssistent the device will be their again and working, because frigate and other integration automatically update their device list. 

So it is a many steps operation that i need to perform every time.

And maybe their is a way to make it more easy...
